extensions-contrib/kubernetes-overlord-extensions/src/test/java/org/apache/druid/k8s/overlord/KubernetesTaskRunnerTest.java

```diff
@@ -263,80 +284,6 @@ public void test_shutdown_withoutExistingTask()
     runner.shutdown(task.getId(), "");
   }
 
-  @Test
-  public void test_restore_withExistingJobs() throws IOException
```

Comment:
why do we need to delete all these tests again?

Reply:
added the missing one back.